.travis.yml

```diff
@@ -5,12 +5,6 @@ compiler:
 sudo: false
 cache: ccache
 
-# blocklist
```

Comment:
"branch isn't used anymore and also it depends
on a private submodule which travis would fail to get it."

->

"branch isn't used anymore and also it used to depend
on a private submodule which travis would fail to get."